src/components/UserInfo/UserInfo.scss

```diff
@@ -0,0 +1,3 @@
+.UserInfo {
+  font-style: italic;
+}
```

Comment:
Consider using a more readable font style than italic. For example, `font-weight: 500` or `font-family: 'Open Sans', sans-serif;`. 📖